### ISSUE TITLE
Restart back-end when Ctrl-D running Circuitpython program, may freeze UI

### ISSUE THREAD

**Issue by aivarannamaa** [thonny/thonny#781]
Try with Metro:

```
from time import sleep

for i in range(1000000):
    sleep(0.01)
    print(i, "jjj")
```

**Comment by aivarannamaa** [comment]
Also try replacing `jjj` with `jjjäöüšš` in the program above and `0.01` with `0.05`

**Comment by aivarannamaa** [comment]
Fixed in 3.2b2
